HW02_A_Graph-Fails.Rmd

```diff
@@ -29,20 +33,33 @@ library("magrittr") #so I can do some piping
 
 What error is being thrown? How do you correct it? (hint, the error message tells you)
 
+**My answer/comment:**
+
+*the following 2 errors have been corrected.*
+
+*1.error 1: use "+"" instead of "%>%"*
+
+*2.error 2: "city" was misspelled; there is a variable called "cty" that means city miles per gallow.*
+
 ```{r}
 data(mpg) #this is a dataset from the ggplot2 package
 
 mpg %>% 
-  ggplot(mapping = aes(x = city, y = hwy, color = "blue")) %>% 
+  ggplot(mapping = aes(x = cty, y = hwy, color = "blue")) +
   geom_point()
 
 ```
 
 ### Graph Fail 2
 Why aren't the points blue? It is making me blue that the points in the graph aren't blue :`(
+
+**My answer/comment:**
+
+*It seems the color command should be wrapped in geom_point(), and my guess is this is because color belongs to this layer instead of ggplot(). I'm not so sure about this.*

```

Comment:
I agree. I think including color="blue" in aes() will add an additional string variable, which only has one value ("blue"), to this layer.